taxcalc/utils.py

```diff
@@ -58,24 +58,24 @@
                       'expanded_income',
                       'aftertax_income']
 
-DIST_TABLE_LABELS = ['Returns',
+DIST_TABLE_LABELS = ['Number of Returns',
```

Comment:
@MaxGhenis See changes to labels on distribution and difference tables for those columns that are counts.  Changed things to "Number of Returns" in both.

cc @andersonfrailey

Reply:
@MaxGhenis and @andersonfrailey Do these labels look fine with you?

Reply:
LGTM, or `# returns` if the table gets too wide with this